contrib/sssd.spec.in

```diff
@@ -159,7 +159,6 @@ Summary: SSSD Client libraries for NSS and PAM
 License: LGPLv3+
 Requires: libsss_nss_idmap = %{version}-%{release}
 Requires: libsss_idmap = %{version}-%{release}
-Requires(post): /sbin/ldconfig
```

Comment:
Why? Because all plugin consumers look at specific locations and ignore ldconfig cache anyway?

Reply:
And `%ldconfig_postun client` is left, this looks inconsistent. Or do I miss something?

Reply:
Actually it needs to stay. I didn't see it is called manually in `%post client`.